doc/classes/PhysicsServer2D.xml

```diff
@@ -61,6 +61,19 @@
 				Creates an [Area2D].
 			</description>
 		</method>
+		<method name="area_enable_shape">
```

Comment:
Same as `enable_shape_owner`, it would be more explicit with something like `area_set_shape_enabled`, without the default value.